README.md

```diff
@@ -1,2 +1,8 @@
 # commands-base
 Stores base commands container configuration for aladdin projects to inherit from
+
+## Examples
+
+1. Using the command-base container image [here](./examples/base_docker_image)
+2. Using a custom container image [here](./examples/custom_docker_image)
+3. Using a custom container image and poetry [here](./examples/poetry_example)
```

Comment:
I had a whole thing written up to expand upon these, but I lost it when i restarted my computer. I could try again. The one example that's kind of missing here is demonstrating that one can use poetry in a commands-base-derived container, but that's not a huge thing.

I think having some context about when each of the options is the best fit would be helpful. I could help write that up if needed.

Reply:
I added a bit more to it, feel free to add some more